modules/reference/pages/images/images-image-list.adoc

```diff
@@ -1,33 +1,26 @@
 [[ref.webui.images.list]]
 = Image List
 
-If you click menu:Main Menu[Images > Images], an overview listing of your images appears.
-Several columns provide information about each image:
 
+The menu:Images[Image List] section displays your current operating system images.
 
-image::images_list_images.png[scaledwidth=80%]
+Click btn:[Import] to import a new image.
```

Comment:
Importing of Kiwi images is not supported. It is valid only for Docker based images. Shouldn't it be mentioned here?